CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+Guidelines on how to contribute to KubeSlice
+
+There are a number of ways you can contribute to the KubeSlice project. We would like to accept your patches and contributions to the KubeSlice code base. This document provides guidelines on how to contribute to the KubeSlice code base. we require all participants to adhere to the [Code of Conduct](https://github.com/kubeslice/worker-operator/blob/doc/contributing-guide/code_of_conduct.md).
+
+
+Here are some ways to contribute:
+
+* Contribute to the kubeslice codebase
+* Report and triage bugs.
+* Answer questions on KubeSlice discussions forums
+* Write technical documentation and blog posts, for users and contributors.
+* Filing issues
+* Provide feedback
+* Submit PRs for small bug fixes
```

Comment:
End this sentence with a period/full stop.

Don't they require to submit PRs for any features/enhancements? cc @gourishkb

Reply:
@rajendra-avesha can comment on this.